doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1208,6 +1208,110 @@ operations will be supported under the 'v2' resource as well.
 
 ### queryContext [POST /v2/queryContext]
 
+<table style="width: 100%; margin: 12px 0 0 0;">
+<tr><td colspan="2"><strong>Required Parameters</strong></td></tr>
+<tr><td colspan="2">none</td></tr>
+<tr><td colspan="2"><strong>Optional Parameters</strong></td></tr>
+<tr><td style="padding-right: 40px; width: 140px;">limit</td><td>Limit the number of entities to be retrieved</td></tr>
+<tr><td style="padding-right: 40px; width: 140px;">offset</td><td>Skip a number of records</td></tr>
+<tr><td style="padding-right: 40px; width: 140px;">options</td><td>
+This operation supports two options:
+
+<ul>
+  <li><code>count</code>: when used, the total entities number is returned as a HTTP in the response name <code>X-Total-Count</code>.</li>
+  <li><code>cannonical</code>: when used, the response payload used the "cannonical form". See JSON entity representation format section
+  for details.</li>
+</td></tr>
+</table>
+
+The response payload is an Array which contains one object per matching entity. Each entity follows
+the JSON entity representation format (described in a section above).
+
+The payload may contain the following elements (all them optional):
+
+* `entities`: a list of entites to search for. Each entity is represented by a JSON object with the following
```

Comment:
In order to not delay this PR merging, this has been deffered to the issue about queryContext implementation https://github.com/telefonicaid/fiware-orion/issues/1080.

NTC